include/aspect/postprocess/interface.h

```diff
@@ -231,6 +232,28 @@ namespace aspect
         PostprocessorType *
         find_postprocessor () const;
 
+        /**
+         * Go through the list of all postprocessors that have been selected
+         * in the input file (and are consequently currently active) and see
+         * if one of them has the desired type specified by the template
+         * argument. If so, return a pointer to it. If no postprocessor is
+         * active that matches the given type, return a NULL pointer.
+         */
+        template <typename PostprocessorType>
+        bool
+        has_postprocessor_of_type () const;
+
+        /**
+         * Go through the list of all postprocessors that have been selected
+         * in the input file (and are consequently currently active) and see
+         * if one of them has the desired type specified by the template
+         * argument. If so, return a pointer to it. If no postprocessor is
+         * active that matches the given type, return a NULL pointer.
+         */
+        template <typename PostprocessorType>
+        PostprocessorType &
+        get_postprocessor_of_type () const;
```

Comment:
Would you want to deprecate `find_postprocess()` in that case?

Also, the docs still talk about pointers.